docs/networks/node-ops/node-operation/reclaim-disk.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Managing disk space
+description: How to manage the node disk space
+---
+
+As the chain advances, nodes receive chain data and store it on disk.
+Hence, the disk usage of a node keeps increasing gradually over time.
+
+In addition to this, currently nodes also experience an intermittent 30-35% spike in disk usage caused by the compaction process of the Badger database used by the node software.
+
+> The spikes will be eliminated once the Badger database is replaced by the Pebble database in the future.
+
+
+Hence, as a node operator, please make sure to do the following:
+
+1. Provision enough disk space as per the node role (see: [node-provisioning](./node-provisioning.md))
+
+
+2. Setup disk usage monitoring and ensure that the node has enough room to grow and to also accommodate those intermittent spikes.
+
+
+3. If needed, please add more disk space to the node from time to time.
+
+> It highly recommended to setup alerting around disk usage to facilitate timely action and avoid any downtime and subsequent reward slashing for the node.
+
+
+## Reclaiming disk space
+
+### Access, Collection, Consensus and Verification node
```

Comment:
it's probably worth noteing that Access nodes will no longer be able to serve AccessAPI requests for data before the dynamic bootstrap height.

Reply:
oh shoot - totally forgot to mention that ...updating.

Reply:
added a section - Limitation for access node